rust/template/differential_datalog/program.rs

```diff
@@ -787,6 +787,50 @@ pub enum Update<V: Val> {
     Modify{relid: RelId, k: V, m: Box<dyn Mutator<V> + Send>}
 }
 
+impl <V: Val> Debug for Update<V> {
```

Comment:
Why do we need to manually implement `Debug`? What makes it superior to `derive(Debug)`? It's not obvious to me. Can you at least add a comment?

Reply:
Ah, I think it's because of the Box with unformattable stuff inside.